### ISSUE TITLE
Cannot run the debugger

### ISSUE THREAD

**Issue by cpuccino** [JetBrains/godot-support#104]
Getting the following error when trying to debug

```
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet---debug does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.

```

It's trying to run the following command:
```
/usr/local/share/dotnet/dotnet --debug --debugger-agent=transport=dt_socket,server=y,suspend=y,address=127.0.0.1:53547,setpgid=y /Applications/Godot.app/Contents/MacOS/Godot --path /Users/xx/Repos/xx res://Levels/Main.tscn
```

**Comment by van800** [comment]
Hi! Try to use the latest Rider EAP build with newest Godot support plugin. What version of Godot do you use btw?

**Comment by cpuccino** [comment]
Removing .idea seems to have fixed it 🤔 - Happened in both 4.0.x and 4.1 - Monterey M1

**Comment by van800** [comment]
If you still have that .idea in the trash, please share it with me. Thanks!
